.gitignore

```diff
@@ -5,3 +5,5 @@ node_modules
 !.yarn/releases
 !.yarn/sdks
 !.yarn/versions
+
+.DS_Store
```

Comment:
nit: This is a minor git house keeping but system files are better ignored at the user level and project files ignored at the project level. https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files